gui/network/forms.py

```diff
@@ -244,7 +244,8 @@ def clean_int_options(self):
         options = self.cleaned_data.get('int_options')
         if iface and options and iface.startswith('lagg') and re.search(r'\bmtu\b', options):
             raise forms.ValidationError(
-                _('MTU option is not allowed for LAGG interfaces')
+                _('MTU option is not allowed for LAGG interfaces. '
```

Comment:
This is not very clear.  Suggestion:

_('To change the MTU of a LAGG interface, set the MTU of each member network interface.')

Reply:
@wblock does your suggestion agree with the one in the original ticket? https://redmine.ixsystems.com/issues/26340

If so, I'll change to your version, looks better to me too.

Reply:
I think it does agree with the suggestion in the bug.  I'm hesitant to suggest exactly where to go in the GUI because there are likely to be GUI changes ahead.

Reply:
I find it unlikely to have changes in the old UI in that regard.

Reply:
Now I agree with @william-gr more